bin/deploy-stack

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
 op=${1:-create}
-heat stack-$op -f templates/virtual-baremetal.yaml -e env.yaml -e templates/resource-registry.yaml baremetal
+heat stack-$op -f templates/virtual-baremetal.yaml -e templates/resource-registry.yaml baremetal -e env.yaml
```

Comment:
I think we need to make this change in deploy-quintupleo too, but I can do that after this merges.